pkg/rbd/rbd_util.go

```diff
@@ -336,7 +338,66 @@ func getMonsAndClusterID(options map[string]string) (monitors, clusterID string,
 	return
 }
 
-func genVolFromVolumeOptions(volOptions map[string]string, disableInUseChecks bool) (*rbdVolume, error) {
+// isLegacyVolumeID checks if passed in volume ID string conforms to volume ID naming scheme used
+// by the version 1.0.0 (legacy) of the plugin, and returns true if found to be conforming
+func isLegacyVolumeID(volumeID string) bool {
```

Comment:
can this be reused for cephfs?

Reply:
Prefix would be different for CephFS legacy volume, so direct reuse is not possible.

Reply:
then I would like to move this function to utils package and pass prefix  as input

Reply:
CephFS is not checking ID format as well, so it will not be called from CephFS even if moved, should I still move it?

Reply:
then no need to move it to utils.